lldb/tools/lldb-dap/JSONUtils.cpp

```diff
@@ -6,25 +6,21 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include <algorithm>
 #include <iomanip>
 #include <optional>
 #include <sstream>
 #include <string.h>
 
 #include "llvm/ADT/StringRef.h"
-#include "llvm/Support/FormatAdapters.h"
 #include "llvm/Support/FormatVariadic.h"
 #include "llvm/Support/Path.h"
 #include "llvm/Support/ScopedPrinter.h"
 
-#include "lldb/API/SBBreakpoint.h"
-#include "lldb/API/SBBreakpointLocation.h"
 #include "lldb/API/SBDeclaration.h"
+#include "lldb/API/SBStream.h"
 #include "lldb/API/SBStringList.h"
 #include "lldb/API/SBStructuredData.h"
 #include "lldb/API/SBValue.h"
-#include "lldb/Host/PosixApi.h"
```

Comment:
Removing this include breaks lldb windows build because it includes `lldb/Host/windows/PosixApi.h` on windows. That provides the definition for `PATH_MAX`, used later in this file. I fixed it on https://github.com/llvm/llvm-project/commit/19c0a74ad6baa9eb38dbe0a20af7c67999c41821

Reply:
Thanks for fixing! 🙂